dart_flexible_sync/README.md

```diff
@@ -0,0 +1,79 @@
+![Realm](https://github.com/realm/realm-dart/raw/master/logo.png)
+
+[![License](https://img.shields.io/badge/License-Apache-blue.svg)](LICENSE)
+
+## A simple command-line application using [Realm Dart SDK](https://www.mongodb.com/docs/realm/sdk/flutter/#dart-standalone-realm) Flexible Sync with [Atlas App Services application](https://www.mongodb.com/docs/atlas/app-services/).
+This sample demonstrates the usage of Flexible Sync with Realm Dart SDK. 
+Writing to a local realm sends the data automatically to the Atlas Cloud Collection.
+Then the local realm is subscribed only for items matching specific query criterias.
+This causes the local realm to be populated only with specific items matching the filter. 
+All other items will exist only in Atlas cloud collection.
+
+# Realm Dart SDK
+
+The Realm Dart package is `realm_dart` and it is available in [pub.dev](https://pub.dev/packages/realm_dart)
+
+## Environment setup for Realm Dart
+
+* Supported platforms are Windows, Mac and Linux.
+
+* Dart SDK ^2.17.1
+
+# Atlas App Services Configuration Steps
+
+* Create an account in [realm.mongodb.com](https://realm.mongodb.com) - follow the instructions: [Get Started with Atlas](https://www.mongodb.com/docs/atlas/getting-started)
+* Creat a new app following the instruction: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui).
+    For the purpose of this sample you don't need to create an app from a template. You can just create an empty application.
+* Click the button in the blue line above - `Review draft & deploy`.
+* Go to `Authentication Providers` menu from the left panel and make sure the option "Allow users to log in anonymously" is ON.
+    Read [this page](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) for more information about the other types of authentication.
+* Go to `Sync` menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
+    * You can skip button "Create a Schema". Choose "No thanks, continue to Sync".
```

Comment:
```suggestion
    * Don't create a schema. Skip by choosing "No thanks, continue to Sync".
```
I think we should ensure the user don't think they can choose their own schema here